package.json

```diff
@@ -176,12 +177,14 @@
     "json5": "2.2.3",
     "plist": "3.0.5",
     "react": "18.2.0",
+    "react-beautiful-dnd": "^13.1.1",
```

Comment:
Drag&Drop for reordering of mods for REDMod integration, I believe we can re-use that in download queue